habitat/hooks/run

```diff
@@ -1,6 +1,7 @@
 #!/bin/sh
 export HOME=/hab/svc/dcob/data
 export RACK_ENV=production
+export SSL_CERT_FILE="$(hab pkg path core/cacerts)/ssl/certs/cacert.pem"
```

Comment:
👍 

😢  But, yes, until [openssl is built patched to the location of Hab cacerts](https://github.com/habitat-sh/core-plans/pull/653).